tests_recording/README.md

```diff
@@ -25,23 +31,18 @@ all credentials there eg:
 
 ### Command to run
 
-Ensure to remove files and directories for tests what you would like to
-regenerate.
+- Ensure to remove files and directories for tests what you would like to regenerate.
+- you have to have configs
+  - for packit: `~/.config/packit.yaml`
+  - for copr: `~/.config/copr`
 
 ```
-PAGURE_TOKEN=your_token GITHUB_TOKEN=your_token pytest-3 -v tests/testsuite_recording
+pytest-3 -v tests/testsuite_recording
 ```
 
 #### Matrix
 
 Install `requre` from git master branch
 
-- first round
-  - `ogr` - from master branch
-  - `rebasehelper` - version `0.19.0` and above
-- second round for test `for test_version_change_exception`
-  - `ogr` - from master branch
-  - `rebasehelper` - `0.17.1`
-- third round
-  - `ogr` - released (rpm or pypi)
-  - `rebasehelper` - `0.19.0` and above
+- now there are no issues with versions of various libraries
+- In case it happen, just regenerate data with proper versions and add there keys for versions
```

Comment:
first sentence says there are no issues and the second tells what to do in case of issues - that doesn't make any sense :D 

also, what does "proper versions" and "add keys" mean exactly?

Reply:
fixed